src/foam.js

```diff
@@ -100,5 +100,14 @@
   //  delete this.FOAM_FILES;
   };
 
+  this.foam = {
```

Comment:
I think we should move this to lib.js so that it's available regardless of the method used to boot foam. If we concat files.js this file won't make the list.